setup.sh

```diff
@@ -75,6 +75,21 @@ function greenprint {
     echo -e "\033[1;32m${1}\033[0m"
 }
 
+# Check file permission
+check_key_perm () {
+    KEY_FILE=$1
+    KEY_PERM=$(stat -L -c "%a %G %U" $1 | grep -oP '\d+' | head -n 1)
+    echo -e "${KEY_PERM}"
+}
+
+# Check ostree_key permissions
+KEY_PERMISSION_PRE=$(check_key_perm "./key/ostree_key")
+echo -e "${KEY_PERMISSION_PRE}"
+if [[ "${KEY_PERMISSION_PRE}" != "600" ]]; then
+   greenprint "💡 File permissions too open...Changing to r/w only for root user"
```

Comment:
600 is not for root user, but for current tty user.
I think you can say "Changing to 600"